worlds/hk/__init__.py

```diff
@@ -484,6 +510,8 @@ def fill_slot_data(self):
 
         slot_data["notch_costs"] = self.charm_costs
 
+        slot_data["GrubCount"] = self.grub_count
```

Comment:
yeah this should probably be snake case to go with the other options, sorry for flipflopping